test/rosdep_repo_check/apk.py

```diff
@@ -93,6 +114,11 @@ def enumerate_apk_packages(base_url, os_name, os_code_name, os_arch):
                 pkg_url = os.path.join(base_url, pkg_filename)
                 yield PackageEntry(pkg_name, pkg_version, pkg_url, source_name=source_name)
 
+                if 'p' in index_entry:
+                    for d in parse_deps(index_entry['p']):
+                        if d.type is None:
+                            yield PackageEntry(d.name, pkg_version, pkg_url, source_name=source_name)
```

Comment:
You should set the `binary_name` value, which is used to keep track of the actual package name when enumerating ephemeral packages like this.
```suggestion
                            yield PackageEntry(d.name, pkg_version, pkg_url, source_name=source_name, binary_name=pkg_name)
```

Reply:
To elaborate, this will ensure that the dashboard links continue to work correctly: https://github.com/ros/rosdistro/blob/e01c69b6814f5b8da96ff636cc9ddf48f6096361/test/rosdep_repo_check/config.yaml#L51

After a quick test, the dashboard seems to expect binary package names and not aliases.